library/core/qir.qs

```diff
@@ -15,7 +15,7 @@ namespace QIR.Runtime {
             fail "Cannot allocate qubit array with a negative length";
         }
         mutable qs = [];
-        for _ in 0..size-1 {
+        for _ in 0..size - 1 {
```

Comment:
So, looks like you decided to not put spaces around the ".."
The expression now looks like you are subtracting 1 from the range (for whatever meaning one can put into such an interpretation) rather than subtracting 1 from the upper bound.
Anyway, maybe it's just my taste. If majority of stakeholders decided that this is the right way - I do not object, just close this comment.